lib/instances/ScreenGui_spec.lua

```diff
@@ -17,29 +17,30 @@ describe("instances.ScreenGui", function()
 	end)
 
 	describe("DisplayOrder", function()
-		it("should have a value that is a number", function()
+		it("should return a number", function()
 			local instance = Instance.new("ScreenGui")
 			assert.equals(type(instance.DisplayOrder), "number")
 		end)
+	end)
 
-		it("should be settable", function()
+	describe("AbsoluteSize", function()
+		it("should return a Vector2", function()
 			local instance = Instance.new("ScreenGui")
-			instance.DisplayOrder = 2
-			assert.equals(instance.DisplayOrder, 2)
+			assert.equals(typeof(instance.AbsoluteSize), "Vector2")
 		end)
+	end)
 
-		it("should only accept numbers", function()
+	describe("AutoLocalize", function()
+		it("should return a boolean", function()
 			local instance = Instance.new("ScreenGui")
-			assert.has.errors(function()
-				instance.DisplayOrder = "string"
-			end)
+			assert.equals(type(instance.AutoLocalize), "boolean")
 		end)
 	end)
 
-	describe("AbsoluteSize", function()
-		it("should return a Vector2", function()
+	describe("IgnoreGuiInset", function()
+		it("should return a boolean", function()
 			local instance = Instance.new("ScreenGui")
-			assert.equals(typeof(instance.AbsoluteSize), "Vector2")
+			assert.equals(type(instance.IgnoreGuiInset), "boolean")
 		end)
```

Comment:
These all being separate `describe` tests is pretty strange, I don't remember when this happened.

Reply:
Yeah these can kind of all be consolidated into the above "should have properties..." test. I'll do that.